angel_system/data/medical/load_bbn_data.py

```diff
@@ -317,26 +338,41 @@ def bbn_activity_txt_to_csv(task, root_dir, output_dir, label_version):
         video_name = os.path.basename(video_dir)
         if video_name in KNOWN_BAD_VIDEOS:
             continue
-
+        
         print(action_txt_fn)
-        with open(action_txt_fn) as action_f:
-            lines = action_f.readlines()
-
+        # with open(action_txt_fn) as action_f:
+        #     lines = action_f.readlines()
+        
```

Comment:
Looks like there are standard style issues in this file. `black` needs to be run, though I just pushed a fix to correctly ignore the ZED wrapper package recently added. Rebase/merge in master and then make sure you run `black .` in your python env or `./angel-workspace-shell.sh` environment.